README.md

```diff
@@ -1,134 +1,142 @@
-# Arweave Dart SDK
```

Comment:
Why are there whitespace changes from top to bottom here?

Reply:
VSCode markdown auto formatting. Should I discard these?

Reply:
Tried to undo it, but it seems just adding that testing section made whitespace changes.

Reply:
I'd say we better have the document formatted so it doesn't happen again in further changes